nsxt/resource_nsxt_policy_lb_http_monitor_profile.go

```diff
@@ -0,0 +1,207 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/data"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyLBHttpMonitorProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyLBHttpMonitorProfileCreate,
+		Read:   resourceNsxtPolicyLBHttpMonitorProfileRead,
+		Update: resourceNsxtPolicyLBHttpMonitorProfileUpdate,
+		Delete: resourceNsxtPolicyLBHttpMonitorProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":                getNsxIDSchema(),
+			"path":                  getPathSchema(),
+			"display_name":          getDisplayNameSchema(),
+			"description":           getDescriptionSchema(),
+			"revision":              getRevisionSchema(),
+			"tag":                   getTagsSchema(),
+			"request_body":          getLbMonitorRequestBodySchema(),
```

Comment:
would it make sense to group `request` attributes in a sub-clause, and same for response?

Reply:
Not sure if it's necessary since we've being using this format for the corresponding non-policy resource?

Reply:
Good point, if we implement resource migration in the future, it would be easier to keep similar format